### PR TITLE
FE - Set up Storybook

### PR DESCRIPTION
**Pull request by Space-zee**
https://app.clickup.com/t/35jvwp7

### CODE REVIEW

File: frontend/stories/button.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.storybook-button {
```
Comment (ruslanpalagin) [inline frontend/stories/button.css:1]: maybe it's better to remove demo files?
We could add some of our component as an example

**Review by ruslanpalagin — CHANGES_REQUESTED** [review]

**Comment by ruslanpalagin** [comment]
Looks good.
I want to run SB before merge.

File: frontend/stories/button.css (hunk 1 above)
Comment (Space-zee) [reply frontend/stories/button.css]: +

Comment (Space-zee) [reply frontend/stories/button.css]: added switcher component

**Review by ruslanpalagin — APPROVED** [review]
cool!

**Pull request merged**